### DISCUSSION TITLE
[Kavita] Pin items to side-nav

### DISCUSSION THREAD

**Discussion by scare376** [Kareadita/Kavita#3067] [Ideas]
### Idea Description

Right now the side nav by default shows the first 10 items and then you click more to see the full list of items. The items you want there might not be the same order that you would want to have the full list. It would be nice if you could instead pin items to go to the "upper" side nav and those can be organized separately from the full list, similarly to how Plex handles it.

### Idea Category

Web UI

### Duration of Using Kavita

2+ years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Ark1369** [comment]
This please, or simply giving user an option to increase the number of items displayed in Sidebar.

**Comment by majora2007** [comment]
Is this still needed? I ended up adjusting the limit to 15 and you can always reorder what's in that top list.

**Reply by scare376** [reply]
I think it would be more ideal to have a list that is separate from the order of the main list

**Reply by Ark1369** [reply]
If custom won't be possible, I would still like a variable number of options the user can set according to their needs. In libraries with mixed content and lot of users, neat to have it all displayed in Sidebar. Especially with Smart Filters now.

**Reply by daydreamrabbit** [reply]
That's right, we really need a lot of pin functionality.

**Comment by majora2007** [comment]
Can someone provide me a mockup (text will do) that helps me understand the organization? 

For me, I have it where the top has items then the More shows the rest of the items in the order you order them in the Customization screen. I'm not getting the gap between this and what users are asking for.

**Comment by LegendaryLass** [comment]
Have two general asks here:
-When things are hidden from the side-nav, there needs to be a way to view them that's fewer clicks than:
--Go into settings
--FInd customize
--click side-nav
--unhide thing
--go back to main view
--click thing
for instance, just making the things on the customize screen click-able into without unhiding them; I recognize this means you go to the main page which probably tries to render a sidebar
another option would be an "Other" option at the bottom of the sidebar that has cards for all these views you've hidden

-When the side-nav kicks in the "More" hiding, it shouldn't be a hard number, it should be based on window size. If it absolutely must be a hard number, it needs to be higher :)
thanks!

**Comment by Leinad4Mind** [comment]
Ideally would be to have Group Permission so Admins or Moderators would have some permissions and could see some itens on sidenav and users would not see the items toggled out.

At the moment the toggling visibility isnt working at all. I disable and it only disables to my account, it should be quite the opposite, hide to everyone else except me (the admin). But I dont mind to hide to everyone. The way it is now don't make any sense.

**Reply by scare376** [reply]
The side nav is per user

**Reply by Leinad4Mind** [reply]
I see. Read only user can't see costumize options. That confused me. Now I understand the idea. I don't like it, but I understand xD

There seems to have a lack of permissions per group to enable/disable functionalities. Or probably I didn't find where.

**Comment by typhoe** [comment]
Some kind of customization of the side bar would indeed be really nice.
e.g. let us choose the number of items to show (you said it's currently 15, it matches what I see and seems to mean 15 items all categories/library names included), so let us choose the default 15 or increase that number for all items.
thank you
